src/message/__tests__/messageActionSheet-test.js

```diff
@@ -56,61 +60,93 @@ describe('constructActionButtons', () => {
   });
 });
 
-describe('constructHeaderActionButtons', () => {
+describe('constructTopicActionButtons', () => {
+  const stream = eg.makeStream();
+  const streamMessage = eg.streamMessage({ stream });
+  const topic = streamMessage.subject;
+  const streamId = streamMessage.stream_id;
+  const streams = deepFreeze(new Map([[streamMessage.stream_id, stream]]));
```

Comment:
nit: cleaner to say `[stream.stream_id, stream]` -- that makes it more obvious that it's correct, and also more stably correct in case of changes to `streamMessage` or something